src/main/java/io/vertx/core/http/HttpClientOptions.java

```diff
@@ -196,6 +206,8 @@
   private int maxPoolSize;
   private boolean keepAlive;
   private int keepAliveTimeout;
+  private boolean isActiveConnectionTTL;
```

Comment:
I think those should be named differently, i.e in relation with the keep-alive feature, I think that keepAliveTTL is a better choice. Also these should be a single value that is either which means disabled or a positive value that is the TTL value.

Reply:
@vietj I will change `isActiveConnectionTTL` to `keepAliveTTL`.
I kept it as single variable initially and set default value as -1 because int in java cannot be null.
It felt it is sort of confusing.
Please suggest if I should revert back to that or you have something else in mind.